projects/packages/publicize/src/jetpack-social-settings/class-settings.php

```diff
@@ -38,6 +38,22 @@ class Settings {
 		'enabled' => true,
 	);
 
+	/**
+	 * Feature flags. Each item has 3 keys because of the naming conventions:
+	 * - flag_name: The name of the feature flag for the option check.
+	 * - plan_name: The name of the plan that enables the feature. Will be checked with Current_Plan.
+	 * - variable_name: The name of the variable that will be used in the front-end.
+	 *
+	 * @var array
+	 */
+	const FEATURE_FLAGS = array(
+		array(
+			'flag_name'     => 'editor_preview',
+			'plan_name'     => 'editor-preview',
```

Comment:
We don't have a plan called 'editor-preview'. This has the potential to be confusing in the future.